lib/matplotlib/backends/backend_wx.py

```diff
@@ -1236,6 +1236,7 @@ def new_figure_manager_given_figure(num, figure):
     figmgr = frame.get_figure_manager()
     if matplotlib.is_interactive():
         figmgr.frame.Show()
+        figure.canvas.draw_idle()
```

Comment:
@mdehoon convinced me that in almost all cases we want to use `draw_idle` to play nice with the crankier event loops (and most of the backends have some 'I have been told to draw, but have not' so the later calls to `draw_idle` get no-oped, rather than stacking up draw commands.